backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -18,6 +22,7 @@ body {
 }
 
 .content-wrapper {
+  background-color: $color-1;
```

Comment:
Could we add a `$content-wrapper-bg-color` variable instead of using a color variable here directly?